### PR TITLE
[README] Rewrite the contributors section

### PR DESCRIPTION
**Pull request by iHDeveloper**
# Information
When I tried to set up the development environment. I ran into issues and complications that took a long time (😓 ). Special thanks to @Berisan (😄 ) for helping me solve those issues.

So, This PR focuses on improving the **Contributors** section in [README](https://github.com/BiscuitDevelopment/SkyblockAddons/blob/development/README.md#for-contributors) by rewriting it.

The section is rewritten in a way that makes it simple and let the contributor focuses on contributing to the project. Instead of figuring out how to run and set up the project.

Also, This PR is part of #338.

# Getting Started
Just visit the `README.md` in my fork by [clicking here](https://github.com/iHDeveloper/SkyblockAddons/blob/development/README.md#for-contributors).

# Checklist
- [x] Rewrite the **Contributors** section
- [x] (Test) Follow the instructions using **IntelliJ IDEA** (Tested by @iHDeveloper)
- [x] (Test) Follow the instructions using **Eclipse** (Tested by @Charzard4261)

**EDIT:** I'm sorry. I didn't understand the note in the contributors' section. Which is not to use the `development` branch in the PR unless it's an important bug fix. So, I prefer to merge this **PR** as fast as possible to avoid any conflicts on it. 😅 

~**EDIT:** If you did the test for **Eclipse**. Post a reply including any issues or problems you had while testing. Also, include a screenshot of the eclipse workspace with the client running.~

### CODE REVIEW

**Comment by iHDeveloper** [comment]
**Special Thanks** to @Charzard4261 for testing using **Eclipse**. 😄 
image
[Source](https://discordapp.com/channels/450878205294018560/596050122438344735/734755403878891601)

# Issues during the test
- Eclipse read files as `Windows-1252` instead of `UTF-8`
 image
 [Source](https://discordapp.com/channels/450878205294018560/596050122438344735/734745949976526959)

**Comment by iHDeveloper** [comment]
I think the contributors' section is ready to be updated 😁

File: README.md [hunk 1]
```diff
@@ -62,26 +62,79 @@ Big thanks to all the open source software we use in this project! They are all
...
+      - Check ☑ `Enable annotation processing`.
+  - [Eclipse](https://projectlombok.org/setup/eclipse)
+
+### Getting started
+**Note:** You can use `gradle` instead using gradle wrapper `./gradlew`.
+
+1. Clone the repository
+2. Setup the development environment
+    ```shell script
+    ./gradlew setupDevWorkspace setupDecompWorkspace
```
Comment (Berisan) [inline README.md:93]: The `setupDevWorkspace` task is included in `setupDecompWorkspace`.
```suggestion
    ./gradlew setupDecompWorkspace
```

File: README.md [hunk 2]
```diff
@@ -62,26 +62,79 @@ Big thanks to all the open source software we use in this project! They are all
...
+    - **After installing the plugin**
+      - Go to `Preferences`
+      - Go to `Build, Execution, Deployment`
+      - Go to `Compiler`
+      - Go to `Annotation Processors`
+      - Check ☑ `Enable annotation processing`.
+  - [Eclipse](https://projectlombok.org/setup/eclipse)
+
+### Getting started
+**Note:** You can use `gradle` instead using gradle wrapper `./gradlew`.
```
Comment (ILikePlayingGames) [inline README.md:88]: Please change to "instead of using the gradle wrapper"

**Comment by ILikePlayingGames** [comment]
Also, do you guys think this should be in a dedicated CONTRIBUTING.md or kept in the main file?

File: README.md (hunk 2 above)
Comment (iHDeveloper) [reply README.md]: Done (f3cf4da) 👍

**Comment by iHDeveloper** [comment]
@ILikePlayingGames Good point. The section became too long and anyone how reads the readme file will be distracted by how to contribute to the project instead of what the project is. I would prefer to separate it in that case.

**Pull request merged**